src/mame/misc/atronic.cpp

```diff
@@ -1,9 +1,72 @@
 // license:BSD-3-Clause
 // copyright-holders:David Haywood
-/* Atronic Russian Video Fruit Machines */
+/* Atronic Video Fruit Machines */
 /*
  From 1999? (documentation is dated August 99)
 
+Atronic was a Austrian/German slot machine manufacturer. It later
+merged with Spielo.
+
+Atronic CashLine platform (~199x-2004?) - z80-based
+
+Atronic CashLine platform () is used in multiple cabinet form-factors:
+* Regular/Upright (?)
+* WBC - Wide Body Cabinet
+* AST - Atronic Slant Top
+* Titan (?)
+
+* CashLine hardware description:
+
+Backplane has room for 4 boards:
+1. Master board [z80]
+2. Graphic Board [TMS34020]
+3. Sound Board (optional?) [YMZ280B?]
+4. Communications board (optional) [m68k]
+
+CashLine PCB boards have EEPROM/EPROM slots for software installation:
+* Main software         - Master board, socket U2
+* Paytable software     - Master board, socket U6
+* Security device (PLA) - Master board, socket U35 (PLA)
+* Graphic software      - Graphic board, sockets U8 .. U15
+* Sound software        - Sound board, sockets U18 .. U21
+* Comm software         - Comm board, sockets U34 and U35
+
+Atronic platforms following z80-based CashLine:
+* "Hi(!)bility" platform (2001) - two 68k-based circuit boards for game control and external
+  communications running OSE (Enea?) operating system, and a customized PC mainboard
+  with Intel Celeron 566MHz running Windows CE and ATI Radeon Mobility M6 AGP graphics
+  card with dual outputs.
```

Comment:
Ideas about the chipset used here?

Reply:
Possibly Intel 815, based on this auction: https://www.ebay.com/itm/266108902344.

Other chipsets I picked up from Dissauer's Masters thesis: [Real-Time 3D Rendering for the Atronic EGD Framework](https://diglib.tugraz.at/download.php?id=577a020e7338b&location=browse)